dozer-sql/expression/src/scalar/string.rs

```diff
@@ -283,11 +477,12 @@ mod tests {
     fn test_string() {
         proptest!(
             ProptestConfig::with_cases(1000),
-            move |(s_val in ".+", s_val1 in ".*", s_val2 in ".*", c_val: char)| {
+            move |(s_val in ".+", s_val1 in ".*", s_val2 in ".*", c_val: char, i_val: i64) | {
                 test_like(&s_val, c_val);
                 test_ucase(&s_val, c_val);
                 test_concat(&s_val1, &s_val2, c_val);
                 test_trim(&s_val, c_val);
+                test_chr(i_val);
```

Comment:
Adding this here will significantly decrease the amount of times the above tests run, because of the increased search space, with the same amount of cases.